Add orgmode support through pandoc

mrvdb opened the pull request:
Quick addition of orgmode through pandoc.

I created this just for myself, feel free to not merge, as pandoc is a bit heavy I guess (?)

mrvdb committed:
Add orgmode support through pandoc

Quick addition of orgmode through pandoc.

snoyberg commented:
How about making it conditional on the presence of a cabal flag?

mrvdb commented:
Sounds good, I have no idea how that would work, but I can find out obviously. (haskell newb here)

snoyberg commented:
In case it's useful, pandoc itself has an example of such conditional compilation: https://github.com/jgm/pandoc/blob/master/pandoc.cabal#L230

mrvdb commented:
Something like this? https://gist.github.com/mrvdb/967d3f2e57d876bd6f93

snoyberg commented:
Something like that, though you'll have to define the CPP variable in the
cabal file.

On Tue, Dec 22, 2015 at 6:15 PM, Marcel van der Boom <
notifications@github.com> wrote:

> Something like this? https://gist.github.com/mrvdb/967d3f2e57d876bd6f93
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/snoyberg/servius/pull/1#issuecomment-166663041.

mrvdb committed:
Make using orgmode through pandoc optional

Use cabal flag orgmode-support which leads
to ORGMODE_SUPPORT being defined in the preprocessor.

mrvdb committed:
Correct typo
